File: EAU.Signing/Models/BtrustUserInputRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EAU.Signing.Models
{
    public class BtrustUserInputRequest
    {
        public string Input { get; set; }

        public BtrustUserInputTypes? InputType { get; set; }

        public string Otp { get; set; }
    }

    public enum BtrustUserInputTypes
    {
        EGN = 0,

        LNCH = 1,

        PROFILE = 2,

        PHONE = 3,

        EMAIL = 4
    }
}

File: EAU.Web.Portal.App/client-app/src/eau-core/models/sign/BtrustUserInputRequest.ts
import { TypeSystem } from 'cnsys-core';
import { moduleContext } from '../../ModuleContext';

export interface BtrustUserInputRequest {
    input: string;
    inputType?: BtrustUserInputTypes;
    otp?: string;
}

export enum BtrustUserInputTypes {
    EGN = 0,
    LNCH = 1,
    PROFILE = 2,
    PHONE = 3,
    EMAIL = 4
}
TypeSystem.registerEnumInfo(BtrustUserInputTypes, 'BtrustUserInputTypes', moduleContext.moduleName);